README.md

```diff
@@ -1,117 +1,69 @@
 
-The following platforms will eventually be included in the "Core" set when properly stabilized as build platforms (i.e. the yaks are bald):
+**[NodeSource](https://nodesource.com/)**, a Node.js enterprise products and services company, donate hardware and hosting for most of the Node.js project's ARM test and build infrastructure.
 
-  * CentOS 5 64-bit (EL5)
-  * CentOS 5 32-bit (EL5)
-  * SmartOS
-  * ARMv6 32-bit (Linux)
-  * ARMv7 32-bit (Linux)
-  * ARMv8 32-bit (Linux, one day, when suitable hardware & OS is available)
-  * ARMv8 64-bit (Linux, one day, when suitable hardware & OS is available)
-  * Ubuntu 14.10 (Utopic Unicorn) 64-bit (preferably prior to official release)
-  * Mac OS X 10.10 (Yosemite) + XCode 6
-  * PowerPC LE (Linux)
-  * PowerPC BE (Linux)
-  * AIX
-
-### Configurations: Non-core
-
-The following platforms will eventually be included in the build and **test-all** set for core-contributors but failure will be advisory-only as they do not constitute core release targets.
-
-  * FreeBSD stable/9 *(maybe)*
-  * FreeBSD stable/10
-  * MinGW 32-bit
-  * MinGW 64-bit
+**[CloudFlare](https://www.cloudflare.com/)**, a CDN and internet traffic management provider, are responsible for providing fast and always-available access to [nodejs.org](https://nodejs.org).
 
+**[ARM](https://www.arm.com/)**, semiconductor intellectual property supplier, have donated ARMv8 hardware for use by the Node.js CI system for build and testing Node.js.
 
 CI Software
 -----------
 
-Build and test orchestration is performed by [Jenkins](http://jenkins-ci.org). You can find a summary of build status [here](https://ci.nodejs.org).
-
-Our ambition is to invest in io.js-specific CI infrastructure and tooling while leaning on existing, proven technologies where appropriate. We hope to slowly replace Jenkins as orchestrator and build-slaves throughout our CI ecosystem.
-
+Build and test orchestration is performed by [Jenkins](http://jenkins-ci.org).
 
-Hardware Sponsors
------------------
-
-The following companies are contributing hardware to this project:
-
-* [DigitalOcean](http://digitalocean.com/)
-* [Rackspace](http://rackspace.com/)
-* [Joyent](http://joyent.com/)
-* [Linaro](http://linaro.org/)
-* [Scaleway](https://www.scaleway.com/)
-* [NodeSource](https://nodesource.com/)
-* [IBM](https://ibm.com) through [OSU Open Source Lab](http://osuosl.org/services/powerdev)
-* [IBM](https://ibm.com) through [Softlayer](http://www.softlayer.com)
+* A summary of build and test jobs can be found at: <https://ci.nodejs.org>
+* A listing of connected servers for testing, building and benchmarking can be found at: <http://ci.nodejs.org/computer/>
 
 
 People
 ------
 
-* Rod Vagg [@rvagg](https://github.com/rvagg)
-* Ben Noordhuis [@bnoordhuis](https://github.com/bnoordhuis)
-* Ryan Graham [@rmg](https://github.com/rmg)
```

Comment:
I still consider myself a member and I'd like to remain on the list, if there aren't any objections. Regular involvement is one of the sources of light at the end of the acquisition/integration tunnel I've been looking forward to.